scripts/qa_crowbarsetup.sh

```diff
@@ -2572,12 +2579,13 @@ function custom_configuration
             fi
 
             if [ -n "$want_sles12" ] && [ -n "$want_docker" ] ; then
-                proposal_set_value nova default "['deployment']['nova']['elements']['${role_prefix}-compute-docker']" "['${unclustered_sles12plusnodes[0]}']"
+                local computenodes=($(get_non_controller_nodes))
+                proposal_set_value nova default "['deployment']['nova']['elements']['${role_prefix}-compute-docker']" "['$computenodes']"
```

Comment:
I think calling the method in this way `get_non_controller_nodes` will fix the problem with the array.